docs/pages/admin-guides/access-controls/device-trust/device-management.mdx

```diff
@@ -82,11 +82,8 @@ Asset Tag    OS    Enroll Status Device ID
 ```
 
 <Admonition type="warning" title="Device Role">
-  For clusters created after v13.3.6, Teleport supports preset `device-admin` role to manage devices.
-
-  For clusters created using Teleport v12 or newer, the preset `editor` role has
-  the necessary permissions to manage devices. For clusters created before v12, you may want
-  to create a dedicated [`device-admin`](#dedicated-device-admin-role) role.
+  For clusters created after v13.3.6, Teleport supports preset `device-admin` role to manage devices
```

Comment:
Might as well remove this too, v13 is EOL.

Reply:
thanks, please see update